### ISSUE TITLE
Improve the ReadME page

### ISSUE THREAD

**Issue by amittri1025** [amittri1025/Whatsapp-Link-Gen#6]
ReadMe Page could be more standarized like other open source repo. If anyone interested. Kindly comment below.

Check for reference.
https://github.com/origranot/reduced.to

**Comment by amittri1025** [comment]
@mounika7204 would you like to work upon this

**Comment by mounika7204** [comment]
@amittri1025  I would like to

**Comment by amittri1025** [comment]
@mounika7204 assigned.

**Comment by amittri1025** [comment]
#15 Done. But it still open for More Creativity 🤩

**Comment by Priyanshusharmaaa** [comment]
Can i contribute too.

**Comment by amittri1025** [comment]
Yes @Priyanshusharmaaa show your Creativity by Contributing to README page. 
Make sure you post Screenshot with your PR.
